scala-package/examples/pom.xml

```diff
@@ -52,6 +52,69 @@
     </profile>
   </profiles>
 
+  <build>
```

Comment:
to generate the lib folder under examples/target/classes after compilation. 
Because a previous pr remove thest lines which cause the lib folder missing after  compilation.
And run some of the examples needs these libs.